README.md

```diff
@@ -20,6 +16,75 @@
+| Link Checker | [![Link Checker][link-check-badge]][link-check-link]            |
+
+### Issues
+
+|                                                                |
+| -------------------------------------------------------------- |
+| [![good first issues open][good-first-badge]][good-first-link] |
+| [![features open][feature-badge]][feature-link]                |
+| [![enhancements open][enhancement-badge]][enhancement-link]    |
+| [![bugs open][bug-badge]][bug-link]                            |
+
+[dco-badge]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/dco.yml/badge.svg
+[dco-badge-link]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/dco.yml
+[link-check-badge]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/link-checker.yml/badge.svg
+[link-check-link]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/link-checker.yml
+[dashboard-build-badge]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/dashboards-reports-test-and-build-workflow.yml/badge.svg
+[dashboard-build-link]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/dashboards-reports-test-and-build-workflow.yml
+[opensearch-build-badge]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/reports-scheduler-test-and-build-workflow.yml/badge.svg
+[opensearch-build-link]: https://github.com/opensearch-project/dashboards-reports/actions/workflows/reports-scheduler-test-and-build-workflow.yml
+[dashboard-codecov-badge]: https://codecov.io/gh/opensearch-project/dashboards-reports/branch/main/graphs/badge.svg?flag=dashboards-reports
+[opensearch-codecov-badge]: https://codecov.io/gh/opensearch-project/dashboards-reports/branch/main/graphs/badge.svg?flag=reports-scheduler
+[codecov-link]: https://codecov.io/gh/opensearch-project/dashboards-reports
+[cypress-test-badge]: https://img.shields.io/badge/Cypress%20tests-in%20progress-yellow
+[cypress-test-link]: https://github.com/opensearch-project/opensearch-build/issues/1124
+[cypress-code-badge]: https://img.shields.io/badge/Cypress%20code-blue
+[cypress-code-link]: https://github.com/opensearch-project/dashboards-reports/tree/main/dashboards-reports/.cypress/integration
+[opensearch-it-badge]: https://img.shields.io/badge/OpenSearch%20Plugin%20IT%20tests-in%20progress-yellow
+[opensearch-it-link]: https://github.com/opensearch-project/opensearch-build/issues/1124
+[opensearch-it-code-badge]: https://img.shields.io/badge/OpenSearch%20IT%20code-blue
+[opensearch-it-code-link]: https://github.com/opensearch-project/dashboards-reports/blob/main/reports-scheduler/src/test/kotlin/org/opensearch/observability/ObservabilityPluginIT.kt
+[bwc-tests-badge]: https://img.shields.io/badge/BWC%20tests-in%20progress-yellow
+[bwc-tests-link]: https://github.com/opensearch-project/dashboards-reports/issues/240
```

Comment:
you may change this, since the PR has just been merged in
https://github.com/opensearch-project/dashboards-reports/pull/244

Reply:
Should I link this to the #244 files for now? Once, the bwc tests start running on Jenkins' distribution builds we can link to it.

Reply:
yeah linking to the files for now makes sense to me.